docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -0,0 +1,184 @@
+
+scrape_configs:
+  - job_name: "prometheus"
+    static_configs:
+      - targets: ["localhost:12345"]
+
+remote_write:
+  - name: "grafana-cloud"
+    url: "https://prometheus-us-central1.grafana.net/api/prom/push"
+    basic_auth:
+      username: USERNAME
+      password: PASSWORD
+```
+
+The convert command takes the YAML as input and outputs a River file.
+
+```bash
+grafana-agent convert --format=prometheus --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+```
+
+The new flow configuration file looks like this:
+
+```river
+prometheus.scrape "prometheus" {
+	targets = [{
+		__address__ = "localhost:12345",
+	}]
+	forward_to     = [prometheus.remote_write.default.receiver]
+	job_name       = "prometheus"
+	scrape_timeout = "45s"
+}
+
+prometheus.remote_write "default" {
+	endpoint {
+		name = "grafana-cloud"
+		url  = "https://prometheus-us-central1.grafana.net/api/prom/push"
+
+		basic_auth {
+			username = "USERNAME"
+			password = "PASSWORD"
+		}
+
+		queue_config {
+			capacity             = 2500
+			max_shards           = 200
+			max_samples_per_send = 500
+		}
+
+		metadata_config {
+			max_samples_per_send = 500
+		}
+	}
+}
+```
+
+### Debugging
+
+1. If the convert command cannot convert a Prometheus configuration,
+   diagnostic information is sent to `stderr`. You can bypass
+   any non-critical issues and output the flow configuration using a 
+   best-effort conversion by including the `--bypass-errors` flag.
+   
+    > Be aware that the behavior may not match when bypassing errors.
+
+    ```bash
+    grafana-agent convert --format=prometheus --bypass-errors --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+    ```
+
+1. You can also output a diagnostic report by including the `--report` flag.
+
+    ```bash
+    grafana-agent convert --format=prometheus --report=OUTPUT_REPORT_PATH --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+    ```
+
+    * Replace `OUTPUT_REPORT_PATH` with the output path for the report.
+
+    Using the example Prometheus configuration from above the diagnostic report provides the following information:
+
+    ```
+    (Info) Converted scrape_configs job_name "prometheus" into...
+      A prometheus.scrape.prometheus component
+    (Info) Converted 1 remote_write[s] "grafana-cloud" into...
+      A prometheus.remote_write.default component
+    ```
+
+## Run a Prometheus configuration
+
+If you’re not ready to switch to a flow configuration, you can also run your
+existing Prometheus configuration without saving it as a flow configuration.
+This allows you to try flow mode without modifying your existing
+Prometheus configuration infrastructure.
+
+> In this task, we will use the [run][] CLI command to run Grafana Agent in flow
+> mode using a Prometheus configuration.
+
+[Start the agent][] in flow mode and include the command line flag
+   `--config.format=prometheus`. Your configuration file must be a valid
+   Prometheus configuration file rather than a flow mode configuration file.
+
```

Comment:
I'm missing the CLI example here... but given the wide range of OS envs, I guess it's kind of hard to show how to start Agent with this flag?

Something like this....

> AGENT_MODE=flow BINARY_PATH run --config.format=prometheus CONFIG_FILE

But this is standalone... and doesn't account for Linux service, macOS, Windows etc.

Reply:
I had an example but then @rfratto rightly pointed out I was assuming too much about how they will be running the agent. we could leave an example and say it's just for 1 use case or keep the link to where all the use cases are documented as it is now? WDYT?

Reply:
Yeah when I started adding the suggestion about adding the example I quickly figured out that... starting agent isn't a single command. My only question is... will the reader assume they are supposed to type `grafana-agent convert` or use systemd/brew/etc. The link does give the right info... so leave it as-is. My comment was more an observation.